friends/src/Components/Friend.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import {connect} from 'react-redux';
+
+const Friend = (props) =>{
+    return(
+        <div key = {props.friend.id}>
+            <p>Name: {props.friend.name}</p>
+            <p>Age: {props.friend.age}</p>
+            <p>Email: {props.friend.email}</p>
```

Comment:
Remember our zoom: you have to know what you're passing down as props in order for everything to behave as expected.